deploy/deploy_ai_service_registry.ts

```diff
@@ -0,0 +1,25 @@
+import {HardhatRuntimeEnvironment} from "hardhat/types"
+import {DeployFunction} from "hardhat-deploy/types"
+
+import ContractDeployer from "../utils/deployer"
+
+const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
+    const {deployments, getNamedAccounts} = hre // Get the deployments and getNamedAccounts which are provided by hardhat-deploy
+
+    const {deployer} = await getNamedAccounts() // Fetch named accounts from hardhat.config.ts
+
+    const contractDeployer = new ContractDeployer(deployer, deployments)
+    const controller = await contractDeployer.fetchDeployedController()
```

Comment:
Given that we don't directly interact with the controller by registering contracts, theoretically, we could simply utilize `deployments.get("Controller")`. However, I've retained this part in the script for two reasons: firstly, in case people wish to extend this deployment script later on, and secondly, to maintain consistency with other proxy deployment scripts like [deploy_bonding_votes.ts](https://github.com/livepeer/protocol/blob/delta/deploy/deploy_bonding_votes.ts).